hydra_base/hydra.ini

```diff
@@ -19,10 +19,16 @@ compression_threshold=50000
 
 [mysqld]
 user = root
-password = root
```

Comment:
Please don't change config stuff in the repo unless it's new or for a very good reason.

Reply:
Restored them